### ISSUE TITLE
Any recommendation for faster inference speed?

### ISSUE THREAD

**Issue by BankNatchapol** [karnwatcharasupat/bandit#10]
I'm trying to use bandit on a large dataset to extract speech only. 
Currently, the processing speed is about 1/5 of the data (100 hours of data takes 20 hours to process).
But I believe it should be able to run much faster than this. 
Even after using the maximum batch size (24 in my case), it's still slow. 

Maybe the reason it's slow is that my dataset contains many small WAV files (1-10 seconds in length)?

Do you have any recommendations to make this faster? 
Thank you in advance!  

Current model: 
     dnr-3s-mus64-l1snr-plus.ckpt

**Comment by kwatcharasupat** [comment]
Yeah, that's slower than it should be. Is the sampling rate of your data the same as the model? Using ffmpeg to preprocess it to the same sampling rate will help a lot, since torchaudio resampling is pretty slow. 

Having a lot of small wav files would also definitely slow you down. There's not much we can do with that though. You could move everything into a channel-first format as an npy file (wav is interleaved), but I doubt the speed up will be significant.
